site/components/src/main/java/uk/nhs/hee/web/components/Listing.java

```diff
@@ -0,0 +1,97 @@
+package uk.nhs.hee.web.components;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * An enumeration of Listing Types and its document types & category value-list identifier.
+ *
+ * <p>Note that the listing types should be in sync
+ * with the value-list {@code /content/documents/administration/valuelists/listingtypes}.</p>
+ */
+public enum Listing {
+
+    /**
+     * Blog Listing
+     */
+    BLOG_LISTING("blog", new String[]{"hee:blogPost"}, "blogCategories"),
+
+    /**
+     * Bulletin Listing
+     */
+    BULLETIN_LISTING("bulletin", new String[]{"hee:bulletin"}, "bulletinCategories"),
```

Comment:
@smanivasag So do we not allow the editor to configure the document types from the CMS ?

Reply:
@thojava - As mentioned in the description (below for ref.), we are still going to allow Content Authors to use `documentTypes` to choose `SearchResults` Content Types and for any future Listing Pages which might potentially require Content Authors to choose the Document/Content Types.

>The Document Types field will still be used to let the author to choose the document types for the SearchResults page in order to offer flexibility. In order to avoid confusions for the author, the description of the Document Types field has been updated to Document Types (Applicable for Search Listing Type).

The reason why I have adapted this approach - Earlier, even though we were allowing Content Authors to choose `documentTypes` for Bulletin page, it is the component (class) which controls which categories (`BulletinCategories` resource bundle) to load. I thought I will get both `documentTypes` and the corresponding Categories resource bundle into component (class) control and let the user to choose a user friendly `ListingType` and we do the rest. I thought this strategy isn't error prone (as in if the Content Author choose an inappropriate `documentTypes` for a Bulletin page for instance). Hope this makes sense.

Reply:
@smanivasag Make sense now :)